CHANGELOG.md

```diff
@@ -1,3 +1,135 @@
+## Release (2023-12-19)
+
+API methods and structs were renamed to have the same look and feel and according to user feedback.
+
```

Comment:
Should we also mention here the main ideas that drove the changes? E.g. `Get<something plural>` to `List...`, `Update...` and `PartialUpdate...`

Reply:
Added summary, please check